ADAL/src/cache/ios/ADKeychainTokenCache.m

```diff
@@ -438,112 +476,24 @@ - (BOOL)removeAllForUserId:(NSString * __nonnull)userId
     return YES;
 }
 
-- (BOOL)removeAllForUserId:(NSString *)userId error:(ADAuthenticationError *__autoreleasing  _Nullable *)error
+- (BOOL)wipeAllItemsForUserId:(NSString *)userId error:(ADAuthenticationError *__autoreleasing  _Nullable *)error
 {
     AD_LOG_WARN(nil, @"Removing all items for user.");
     AD_LOG_WARN_PII(nil, @"Removing all items for userId <%@>", userId);
 
-    NSArray *items = [self allItems:nil];
-    if (!items)
-    {
-        return NO;
-    }
+    NSDictionary *query = @{ (id)kSecClass : (id)kSecClassGenericPassword,
```

Comment:
nit: should we check that passed userId is non-nil?

Reply:
sure, i agree.